app/src/main/java/org/p2p/wallet/send/ui/dialogs/SendTransactionsDetailsBottomSheet.kt

```diff
@@ -0,0 +1,157 @@
+        with(itemAddress) {
+            root.isVisible = !addressIsEmpty
+            imageViewIcon.setImageResource(R.drawable.ic_wallet_home)
+            textViewTitle.text = getString(R.string.send_transactions_details_address)
+            textViewSubtitle.text = address
+            itemAddress.root.setOnLongClickListener {
+                requireContext().copyToClipBoard(address.orEmpty())
+                true
+            }
+        }
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setRecipientGets() {
+        val color = getColor(R.color.text_mountain)
+        with(itemGets) {
+            imageViewIcon.setImageResource(R.drawable.ic_receive)
+            textViewTitle.text = getString(R.string.send_transactions_details_gets)
+            textViewSubtitle.text = SpanUtils.highlightText(
+                state.fullReceive,
+                state.approxReceive,
+                color
+            )
+        }
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setTransactionFee() {
+        val colorNight = getColor(R.color.text_mountain)
+        val colorMountain = getColor(R.color.text_night)
+        val colorMint = getColor(R.color.text_mint)
+        with(itemTransactionFee) {
+            imageViewIcon.setImageResource(R.drawable.ic_lightling)
+            textViewTitle.text = getString(R.string.send_transactions_details_transaction_fee)
+            textViewSubtitle.apply {
+                when (val fee = state.fee) {
+                    is SendFee.SolanaFee -> {
+                        setTextColor(colorNight)
+                        text = SpanUtils.highlightText(
+                            fee.accountCreationFullFee,
+                            fee.approxAccountCreationFeeUsd.orEmpty(),
+                            colorMountain
+                        )
+                    }
+                    else -> {
+                        setTextColor(colorMint)
+                        setText(R.string.send_free_transaction)
+                        // TODO PWN-6092 make counting of free transactions and set it here!
+                    }
+                }
+            }
+        }
+    }
+
+    private fun DialogSendTransactionsDetailsBinding.setAccountCreation() {
+        val colorNight = getColor(R.color.text_mountain)
+        val colorMountain = getColor(R.color.text_night)
+        val colorMint = getColor(R.color.text_mint)
+
+        groupAccountFee.isVisible = state.showAccountCreation
+        imageViewAccountFeeInfo.setImageResource(R.drawable.ic_user)
+        textViewTitleAccountFee.text = getString(R.string.send_transactions_details_account_fee)
+        textViewSubtitleAccountFee.text = getString(R.string.send_transactions_details_account_fee)
+
+        textViewSubtitleAccountFee.apply {
+            when (val fee = state.fee) {
+                is SendFee.SolanaFee -> {
+                    setTextColor(colorNight)
+                    text = SpanUtils.highlightText(
```

Comment:
HighlightText updates the text to black color.
Maybe we need to update it to lime?